server/app/featureflags/FeatureFlags.java

```diff
@@ -18,64 +30,41 @@
  */
 public final class FeatureFlags {
   private static final Logger logger = LoggerFactory.getLogger(FeatureFlags.class);
-  // Main control for any feature flags working.
-  private static final String FEATURE_FLAG_OVERRIDES_ENABLED = "feature_flag_overrides_enabled";
-
-  // Long lived feature flags.
-  public static final String ALLOW_CIVIFORM_ADMIN_ACCESS_PROGRAMS =
-      "allow_civiform_admin_access_programs";
-  public static final String ADMIN_REPORTING_UI_ENABLED = "admin_reporting_ui_enabled";
-  public static final String SHOW_CIVIFORM_IMAGE_TAG_ON_LANDING_PAGE =
-      "show_civiform_image_tag_on_landing_page";
-
-  // Launch Flags, these will eventually be removed.
-  public static final String PROGRAM_ELIGIBILITY_CONDITIONS_ENABLED =
-      "program_eligibility_conditions_enabled";
-  public static final String PROGRAM_READ_ONLY_VIEW_ENABLED = "program_read_only_view_enabled";
-
   private final Config config;
 
-  // Address correction and verifcation flags
-  private static final String ESRI_ADDRESS_CORRECTION_ENABLED = "esri_address_correction_enabled";
-  private static final String ESRI_ADDRESS_SERVICE_AREA_VALIDATION_ENABLED =
-      "esri_address_service_area_validation_enabled";
-
-  // Common Intake Form flags.
-  private static final String INTAKE_FORM_ENABLED = "intake_form_enabled";
-  public static final String NONGATED_ELIGIBILITY_ENABLED = "nongated_eligibility_enabled";
-
-  // phone number question type
-  private static final String PHONE_QUESTION_TYPE_ENABLED = "phone_question_type_enabled";
-
   @Inject
   FeatureFlags(Config config) {
     this.config = checkNotNull(config);
   }
 
   public boolean areOverridesEnabled() {
-    return config.hasPath(FEATURE_FLAG_OVERRIDES_ENABLED)
-        && config.getBoolean(FEATURE_FLAG_OVERRIDES_ENABLED);
+    return config.hasPath(FeatureFlag.FEATURE_FLAG_OVERRIDES_ENABLED.toString())
+        && config.getBoolean(FeatureFlag.FEATURE_FLAG_OVERRIDES_ENABLED.toString());
   }
 
   /**
    * If the Eligibility Conditions feature is enabled.
    *
    * <p>Allows for overrides set in {@code request}.
    */
+  // TODO(#4437): remove and have clients call getFlagEnabled directly.
```

Comment:
isn't the PR #4437? Did we mean #4447 here and elsewhere? (also _very_ nit but s/remove/Remove/)

Reply:
Yep, thanks.